Files/Views/InstanceTabsView.xaml.cs

```diff
@@ -131,6 +134,11 @@ public async void AddNewTab(Type t, string path)
                     tabLocationHeader = ResourceController.GetTranslation("SidebarVideos");
                     fontIconSource.Glyph = "\xE8B2";
                 }
+                else if (path.StartsWith(App.AppSettings.RecycleBinPath, StringComparison.OrdinalIgnoreCase))
+                {
+                    tabLocationHeader = ApplicationData.Current.LocalSettings.Values.Get("RecycleBin_Title", "Recycle Bin");
```

Comment:
What about to use localSettings.Values?

Reply:
Shouldn't we use `App.AppSettings`?

Reply:
Perhaps it would be better just to add the localized recycle bin name to the translation files?